### PR TITLE
V0.6.2

### PR DESCRIPTION
**Pull request by oguzhanunlu**
This PR updates how Caddy is installed, configured and used. Also fixes the related bug in 0.6.1 release.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,93 +0,0 @@
-github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```
Comment (BenFradet) [inline go.sum]: do we need these files after all or no?

File: provisioning/resources/init/caddy [hunk 2]
```diff
@@ -1,98 +1,109 @@
...
+DAEMON_LOG='/var/log/caddy.log'
 
-get_pid() {
-    cat "$pid_file"
+[ -r "/etc/default/${DAEMON_NAME}" ] && . "/etc/default/${DAEMON_NAME}"
+
+do_start() {
+  local result
+
+	pidofproc -p "${DAEMON_PID}" "${DAEMON_PATH}" > /dev/null
```
Comment (BenFradet) [inline provisioning/resources/init/caddy]: indent seems weird

Comment (oguzhanunlu) [reply provisioning/resources/init/caddy]: oh, right

File: go.sum (hunk 1 above)
Comment (oguzhanunlu) [reply go.sum]: we needed them in the previous version, we changed the way caddy is installed, hence the removal

File: go.mod [hunk 3]
```diff
@@ -1,9 +0,0 @@
-module github.com/snowplow/snowplow-mini
-
-go 1.12
-
-require (
-	github.com/jinzhu/inflection v0.0.0-20190603042836-f5c5f50e6090 // indirect
-	github.com/mholt/caddy v1.0.0 // indirect
-	gopkg.in/pg.v5 v5.3.3 // indirect
-)
```
[the remark this answers is not in the document]
Comment (oguzhanunlu) [reply go.mod]: explanation below for both files

**Pull request merged**